contracts/Migrations.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.0;
```

Comment:
Why not apply 0.5.9?

Reply:
It's a workaround to get solidity-coverage working. Although we explicitly set solidity compiler version to 0.5.9 it changes it during generated coverage to 0.5.3.